docs/v1/api.md

```diff
@@ -1,63 +1,96 @@
+catalog.
 
-1. Make a request against a given CF service broker's catalog endpoint
-   (`GET /v2/catalog`)
-2. Translate the response to a list of `ServiceClass`es
-3. Write each translated `ServiceClass` to stable storage
-
-*TODO: should we call out what happens when a `Broker` resource is deleted?*
-
-## `ServiceClass`
-
-This resource is created by the service catalog's controller event loop after
-it has received a `Broker` resource and successfully called the backing CF
-broker's catalog endpoint. It represents a single (serviceID, planID) pair that
-a CF service broker has made available.
+An administrator to creates an instance of the `Broker` resource to indicate
+their intent to consume that broker in the service catalog. 
 
-*TODO: what happens in the below cases?*
+The catalog controller has a watch on the `Broker` resource.  When the
+controller receives an `ADD` watch event for a new `Broker`, it contacts the
+broker to determine what service classes the broker offers:
 
-1. *The `Broker` that caused the `ServiceClass` was deleted*
-1. *The `ServiceClass` itself was deleted*
-
-## `Instance`
-
-This resource is created by a service consumer to indicate their desire to
-provision a service. When the service catalog's controller event loop sees an
-`Instance` created, it calls the provision endpoint on the backing CF service
-broker and writes `provisioned` into a `status` field of the
-`Instance`.
-
-*TODO: what happens when an `Instance` resource is deleted?*
+1. Makes a request against a given CF service broker's catalog endpoint
+   (`GET /v2/catalog`)
+2. Translates the response to a list of `ServiceClass`
+3. Creates new `ServiceClass` instances in the API server
 
-## `Binding`
+*TODO: What happens when a `Broker` resource is deleted?*
 
-This resource is created by a service consumer to indicate that an application
-should be bound to an instance. When the service catalog's controller event
-loop sees a `Binding` created, it calls the bind endpoint on the backing CF
-service broker. When a successful response is returned, the event loop does
-the following:
+## `ServiceClass` Resource
 
-1. Writes `bound` into a `status` field of the `Binding`
-1. Writes the contents of `credentials` map into a secret (naming of the secret
-   to be discussed later)
-1. Updates the `Binding`'s status section with the fully qualified path to the
-   aforementioned secret
+A `ServiceClass` represents an offering in the service catalog.  A
+`ServiceClass` is not usable directly; an `Instance` of a `ServiceClass` must be
+created to be consumed by an application.
 
-*TODO: what happens when a `Binding` resource is deleted?*
+This resource is created by the service catalog's controller event loop after
+it has received a `Broker` resource and successfully called the backing CF
+broker's catalog endpoint.
+
+*TODO: Make explicit the relationship between service classes and plans*
```

Comment:
Not necessarily.  It only means that we did not have an agreement on it and need to discuss as a group.  I have my own opinions but we need to agree as a group.